docs/kernels-overview.md

```diff
@@ -1,38 +1,44 @@
 # .NET Interactive Architectural Overview 
 
-The kernel concept in .NET Interactive is a component that accepts commands and produces outputs. The commands are  typically blocks of arbitrary code, and the outputs are events that describe the results and effects of that code. The `Kernel` class represents this core abstraction.
+The kernel concept in .NET Interactive is a component that accepts commands and produces outputs. The commands are typically blocks of code, and the outputs are events that describe the results and effects of that code, including executing code and providing language services such as completions and diagnostics. The `Kernel` class represents this core abstraction.
```

Comment:
do you mean 'auto-completion' instead of 'completions' ?

Reply:
The LSP terminology is "completions" but I'm not sure which term is more generally understood.